internal/pkg/utils/keybase.go

```diff
@@ -0,0 +1,143 @@
+// Copyright © 2018 Cisco Systems, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/cisco-sso/kdk/internal/pkg/utils/simpleprompt"
+	"github.com/codeskyblue/go-sh"
+)
+
+// Write keybase mirror script [windows only]
+func keybaseWriteMirrorScript(configDir string) (out string, err error) {
+
+	cmdString := `
```

Comment:
SHOULD: make a const and pull out of this function

Reply:
fixed